netbox/netbox/settings.py

```diff
@@ -653,6 +653,7 @@ def _setting(name, default=None):
     'VERSION': VERSION,
     'COMPONENT_SPLIT_REQUEST': True,
     'REDOC_DIST': 'SIDECAR',
+    'SERVE_PUBLIC': getattr(configuration, 'PUBLIC_SWAGGER', True),
```

Comment:
The config parameter should first be defined toward the top of `settings.py` for consistency with other parameters, then referenced here.

Reply:
ack, done in the new commit.